### ISSUE TITLE
No notification showed when task is finished

### ISSUE THREAD

**Issue by Katka92** [eclipse-che/che#14724]
### Describe the bug
When the task is run, user is notified by pop-up like this:
![Screenshot from 2019-10-01 12-39-16](https://user-images.githubusercontent.com/29278800/65955479-84e6b080-e448-11e9-9a75-4b3f507fa868.png)

In previous version (7.1.0) user were notified in a same way when task ended with message similar too ` Task che: maven build has exited with code 0. `. In current version 7.2.0 no pop-up is shown. 

### Che version

 - [ ] latest
 - [x] nightly
 - [x] other: 7.2.0

### Steps to reproduce
1. Create default Java Maven workspace with example project
2. Once workspace is ready, click Terminal -> Run task... -> che: mavne build -> Continue without scanning task output
3. Wait until build finishes
4. No pop-up is shown.

### Expected behavior
The pop-up informing user that task has finished should be shown. 

### Runtime
   - [ ] kubernetes (include output of `kubectl version`) 
   - [ ] Openshift (include output of `oc version`)
   - [x] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Installation method
   - [x] chectl 
   - [ ] che-operator 
   - [ ] minishift-addon
   - [ ] I don't know

### Environment
   - [x] my computer 
       - [ ] Windows
       - [x] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify) 
   - [ ] other: please specify

**Comment by tolusha** [comment]
It seems the task is counted as `running`.

![Screenshot from 2019-10-01 14-17-34](https://user-images.githubusercontent.com/1640675/65957728-435d0200-e456-11e9-8e82-72ff48e3a0c9.png)

**Comment by l0rd** [comment]
@vparfonov @RomanNikitenko is this normal behavior for latest version of Theia?

**Comment by RomanNikitenko** [comment]
I'm going to investigate the cause of the issue.

@vparfonov can we take it in current sprint as unplanned issue?

**Comment by vparfonov** [comment]
i don't think it's so critical, we can wait on next sprint

**Comment by l0rd** [comment]
Yes I agree with @vparfonov. Not critical, can be postponed until next sprint.

**Comment by amisevsk** [comment]
How did an issue like this pass e2e/happy-path tests? I agree that it's not critical, but also would make any command that naturally doesn't have output on success untrackable to the user.

**Comment by RomanNikitenko** [comment]
My current issue is Stop running task https://github.com/eclipse/che/issues/13737,
so I would like to investigate and fix this bug to have ability to test my changes for https://github.com/eclipse/che/issues/13737. 

@l0rd @vparfonov 
can we take it in current sprint as unplanned issue?
